spread.yaml

```diff
@@ -0,0 +1,152 @@
+project: calibre-web-charm-k8s-tests
```

Comment:
Sorry, I committed a bunch of files that were provided by `charmcraft init`, and as far as I can tell `test/spread` isn't used at all at the moment, so I'll remove those files.

Reply:
FYI, the spread files power `charmcraft test`. It's currently very poorly documented and not really clear where this is meant to head (it's on the list for discussion in The Hague).

Reply:
Sorry for the repetition, but -- I had no idea about `charmcraft test`!

I think I'll remove the spread stuff for now since I've just been using `tox` anyway.